miner/currency_blocklist.go

```diff
@@ -18,21 +19,41 @@ type AddressBlocklist struct {
 	// will get evicted when evict() is called
 	headerEvictionTimeoutSeconds uint64
 	oldestHeader                 *types.Header
+
+	enactFilterEnabled *atomic.Int32
```

Comment:
Why not use [`atomic.Bool`](https://pkg.go.dev/sync/atomic#Bool)?

Reply:
That's a very good question... somehow I thought it doesn't exist